openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -292,22 +292,20 @@ def get(self, request):
 
     def list(self, request):
         """
-        GET /api/user/v1/accounts?username={username1,username2}
-        GET /api/user/v1/accounts?email={user_email}
+        GET /api/user/v1/accounts?username={username1},{username2}
+        GET /api/user/v1/accounts?email={user_email1},{user_email2}
         """
         usernames = request.GET.get('username')
-        user_email = request.GET.get('email')
+        user_emails = request.GET.get('email')
         search_usernames = []
 
         if usernames:
             search_usernames = usernames.strip(',').split(',')
-        elif user_email:
-            user_email = user_email.strip('')
-            try:
-                user = User.objects.get(email=user_email)
-            except (UserNotFound, User.DoesNotExist):
+        elif user_emails:
```

Comment:
What if one email is passed from the FE then this strip function will give an error?

Reply:
No, it will not.